go.mod

```diff
@@ -8,26 +8,43 @@ require (
 	github.com/docker/go-connections v0.4.0
 	github.com/ethereum/go-ethereum v1.12.0
 	github.com/go-resty/resty/v2 v2.7.0
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.3.1
 	github.com/jmoiron/sqlx v1.3.5
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/lib/pq v1.10.9
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.16.0
+	github.com/prometheus/client_golang v1.17.0
 	github.com/prometheus/common v0.44.0
 	github.com/rs/zerolog v1.30.0
 	github.com/slack-go/slack v0.12.2
-	github.com/smartcontractkit/chainlink-env v0.37.0
+	github.com/smartcontractkit/chainlink-env v0.38.3
 	github.com/smartcontractkit/wasp v0.3.0
 	github.com/stretchr/testify v1.8.4
 	github.com/testcontainers/testcontainers-go v0.23.0
-	go.uber.org/atomic v1.10.0
-	go.uber.org/zap v1.25.0
-	golang.org/x/sync v0.3.0
+	go.uber.org/atomic v1.11.0
+	go.uber.org/zap v1.26.0
+	golang.org/x/sync v0.4.0
+)
+
+require (
+	github.com/allegro/bigcache v1.2.1 // indirect
+	github.com/cespare/cp v1.1.1 // indirect
+	github.com/gballet/go-libpcsclite v0.0.0-20191108122812-4678299bea08 // indirect
+	github.com/google/pprof v0.0.0-20230705174524-200ffdc848b8 // indirect
+	github.com/hashicorp/go-version v1.6.0 // indirect
+	github.com/mattn/go-sqlite3 v2.0.3+incompatible // indirect
+	github.com/onsi/ginkgo/v2 v2.9.7 // indirect
+	github.com/onsi/gomega v1.27.8 // indirect
+	github.com/pyroscope-io/client v0.7.1 // indirect
+	github.com/rs/cors v1.8.3 // indirect
+	github.com/yusufpapurcu/wmi v1.2.3 // indirect
 )
 
 // K8s versions are infuriatingly inconsistent, so we pin them here.
 replace (
+	// replicating the replace directive on cosmos SDK
```

Comment:
Do you still need these replaces now that you aren't importing chainlink repo? Both this one and the one lower down seem out of place based on the changes you made in the code.

Reply:
@tateexon yep, I'm getting lot of errors otherwise:

```
go: github.com/smartcontractkit/chainlink-testing-framework/logwatch imports
        github.com/smartcontractkit/wasp imports
        github.com/grafana/loki/clients/pkg/promtail/client imports
        github.com/grafana/loki/pkg/util/build imports
        github.com/prometheus/prometheus/web/api/v1 imports
        github.com/prometheus/prometheus/config tested by
        github.com/prometheus/prometheus/config.test imports
        github.com/prometheus/prometheus/discovery/ovhcloud imports
        github.com/ovh/go-ovh/ovh: github.com/gogo/protobuf@v1.3.3: reading github.com/gogo/protobuf/go.mod at revision v1.3.3: unknown revision v1.3.3
go: github.com/smartcontractkit/chainlink-testing-framework/logwatch imports
        github.com/smartcontractkit/wasp imports
        github.com/grafana/loki/clients/pkg/promtail/client imports
        github.com/grafana/loki/pkg/util/build imports
        github.com/prometheus/prometheus/web/api/v1 imports
        github.com/prometheus/prometheus/config tested by
        github.com/prometheus/prometheus/config.test imports
        github.com/prometheus/prometheus/discovery/scaleway imports
        github.com/scaleway/scaleway-sdk-go/api/baremetal/v1: github.com/gogo/protobuf@v1.3.3: reading github.com/gogo/protobuf/go.mod at revision v1.3.3: unknown revision v1.3.3
go: github.com/smartcontractkit/chainlink-testing-framework/logwatch imports
```